src/ontology/mondo-edit.obo

```diff
@@ -62220,7 +62221,7 @@ xref: ICD9:307.49 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MEDGEN:167920 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MESH:D012893 {source="MONDO:equivalentTo"}
 xref: SCTID:39898005 {source="MONDO:equivalentTo"}
-xref: UMLS:C4042891 {source="https://orcid.org/0000-0002-3458-4839"}
```

Comment:
This line should be maintained based on the [ticket comment](https://github.com/monarch-initiative/mondo/issues/7189#issuecomment-2322144956) for this term. I added a [bug report](https://github.com/monarch-initiative/mondo/issues/8170).

The